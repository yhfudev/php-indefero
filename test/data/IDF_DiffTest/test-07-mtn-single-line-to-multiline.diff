#
# old_revision [7950c8a4e13342d1dca94c05efc49b291ab327f5]
#
# patch "a"
#  from [023cc93272e5cbc11f128ffc90fbbe42f6f08d8c]
#    to [b3d40c241ccfd6ff1a06047a9f6d609227d2893f]
#
============================================================
--- a	023cc93272e5cbc11f128ffc90fbbe42f6f08d8c
+++ a	b3d40c241ccfd6ff1a06047a9f6d609227d2893f
@@ -1 +1,2 @@ ls
 ls
+ls
