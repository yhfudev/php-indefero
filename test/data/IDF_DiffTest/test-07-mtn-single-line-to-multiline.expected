<?php return array (
  'a' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '1',
          2 => "ls\r\n",
        ),
        1 => 
        array (
          0 => '',
          1 => 2,
          2 => "ls\r\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => 1,
        ),
        1 => 
        array (
          0 => '1',
          1 => '2',
        ),
      ),
    ),
  ),
);